docs/fix/empty-wheelbarrows.rst

```diff
@@ -0,0 +1,33 @@
+fix/empty-wheelbarrows
+======================
+
+.. dfhack-tool::
+    :summary: Empties stuck items from wheelbarrows
+    :tags: fort bugfix items
+
+Empties all wheelbarrows which contain rocks that have become 'stuck' in them.
+
+This works around the issue encountered with :bug:`6074`, and should be run
+if you notice wheelbarrows lying around with rocks in them that aren't
+being used in a task.
```

Comment:
is there any downside to periodically running this tool in the background? If not, then you should add it to the Maintenance list in `gui/control-panel` and to the default-enabled list in https://github.com/DFHack/dfhack/blob/develop/data/dfhack-config/init/onMapLoad.control-panel-repeats.init

Reply:
The issue it solves is only really an issue if you're like me and use a lot of different stone stockpiles and feeders, if people don't use wheelbarrows but have maybe 3000 jugs or other tools I don't know how performance will be impacted as I haven't measured this. If I wanted to add it there I suppose I'd have to open another PR over on the dfhack develop repo?

Reply:
another PR just for the addition of the default enable in `onMapLoad.control-panel-repeats.init`. `gui/control-panel.lua` is in the scripts repo and can be modified in this PR.